packages/engine/package.json

```diff
@@ -78,7 +78,7 @@
     "@types/offscreencanvas": "2019.6.4",
     "@types/proxyquire": "^1.3.28",
     "@types/sinon": "10.0.11",
-    "@types/three": "0.136.1",
+    "@types/three": "0.138.0",
```

Comment:
Why it is different from `three` version?

Reply:
three types are always a few weeks behind, as it is managed entirely separately from threejs itself. there are no typing updates between these versions, so it is all fine.